cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/cube/CubeExploreQuery.java

```diff
@@ -34,6 +33,13 @@
   private final int limit;
   private final List<TagValue> tagValues;
 
+  /**
+   * Creates instance of {@link CubeExploreQuery} that defines selection of data of {@link Cube} to explore in.
+   * @param startTs start time of the data selection, in seconds since epoch
+   * @param endTs end time of the data selection, in seconds since epoch
+   * @param resolution resolution of the aggregations explore
+   * @param tagValues tag name, tag value pairs that define the data selection
+   */
   public CubeExploreQuery(long startTs, long endTs, int resolution, int limit, List<TagValue> tagValues) {
```

Comment:
Is ordering important for `tagValues`? If not, use more generic form like `Collection` or `Iterable`.